src/client/mod.rs

```diff
@@ -480,13 +481,18 @@ impl Client {
         &self,
         criteria: Option<&SelectionCriteria>,
     ) -> Result<ServerAddress> {
-        let server = self.select_server(criteria).await?;
+        let server = self.select_server(criteria, "test_select_server").await?;
```

Comment:
The names given elsewhere are short phrases rather than a snake_case tag, might be better to be consistent one way or the other.

Reply:
renamed to "Test select server" for consistency.